bin/yii

```diff
@@ -10,42 +10,29 @@ use Yiisoft\Yii\Console\Application;
 use Yiisoft\Yii\Console\Output\ConsoleBufferedOutput;
 
 (static function () {
-    $cwd = getcwd();
-
-    $possibleAutoloadPaths = [
-        // running from project root
-        $cwd . '/vendor/autoload.php',
-        // running from project bin
-        dirname($cwd) . '/autoload.php',
-        // local dev repository
-        dirname(__DIR__) . '/vendor/autoload.php',
-        // dependency
-        dirname(__DIR__, 4) . '/vendor/autoload.php',
-    ];
-
     $autoloadPath = null;
 
-    foreach ($possibleAutoloadPaths as $possibleAutoloadPath) {
+    for ($x = 5; $x > 0; $x--) {
+        $possibleAutoloadPath = dirname(__FILE__, $x) . '/vendor/autoload.php';
         if (file_exists($possibleAutoloadPath)) {
             $autoloadPath = $possibleAutoloadPath;
             break;
         }
     }
 
     if ($autoloadPath === null) {
-        $message = "Unable to find vendor/autoload.php in the following paths:\n\n";
-        $message .= '- ' . implode("\n- ", $possibleAutoloadPaths) . "\n\n";
+        $message = "Unable to find vendor/autoload.php\n\n";
         $message .= "Possible fixes:\n";
         $message .= "- Install yiisoft/console via Composer.\n";
-        $message .= "- Run ./yii either from project root or from vendor/bin.\n";
+        $message .= "- Make sure yii runs from inside your installation folder.\n";
```

Comment:
Maybe `from application root folder`

Reply:
Explicit no, as that is fixed with this PR.

I tried to say that the location of the yii fie must be inside the yii folder. In a default installation it will be `vendor/yiisoft/yii-console/bin/yii`. The wording could better I guess, I just did not find the words :)

Reply:
"Make sure the yii executable is located inside your installation folder." ?

Reply:
Make sure the yii executable is located inside your project folder